userWhitelist.json

```diff
@@ -13,7 +13,7 @@
     "kevintab95",
     "seanlip",
     "ankita240796",
-    "Showtim3",
+    "DubeySandeep",
```

Comment:
Maybe also add Srijan Redy to this list.

Reply:
Done.